NuGet.config

```diff
@@ -4,8 +4,10 @@
   <packageSources>
     <clear />
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+    <add key="general-testing" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/general-testing/nuget/v3/index.json" />
```

Comment:
Ignore for now, will remove when the SBRP update is done.

Reply:
Can this be removed now?